_src/static/sass/base/_variables.scss

```diff
@@ -0,0 +1,161 @@
+/* ============================
+-------------------------------
+ 1. Grid
+ 2. Fonts
+ 3. Colors (Palette)
+ 4. Colors (Application)
+ 5. Timing
+-------------------------------
+============================ */
+
+$baseline:                      20px;
+
+
+/* ============================
+1. Grid
+---------------------------- */
+$gw-column:                     ($baseline*3);
+$gw-gutter:                     $baseline;
+$fg-column:                     $gw-column;
+$fg-gutter:                     $gw-gutter;
+$fg-max-columns:                12;
+$fg-max-width:                  1600px;
+$fg-min-width:                  768px;
+
+
+/* ============================
+2. Fonts
+---------------------------- */
+$font-sans:                    'Open Sans','Helvetica Neue', Helvetica, Arial, sans-serif;
```

Comment:
NIT: alignment of values is nice, but this is a ton of whitespace between all names/values. Please tighten.